Code/GraphMol/Descriptors/Lipinski.cpp

```diff
@@ -469,5 +472,30 @@ unsigned int calcNumBridgeheadAtoms(const ROMol &mol,
   return atoms->size();
 }
 
+const std::string NumAtomStereoCentersVersion = "1.0.0";
+unsigned int numAtomStereoCenters(const ROMol &mol) {
```

Comment:
Since this requires that the chirality perception code have been run, it may be a good idea to call `mol.hasProp(common_properties::_StereochemDone))` and, if not, call `assignStereochemistry()`